test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2104,6 +2160,9 @@ class test_ucp_sockaddr_protocols : public test_ucp_sockaddr {
         bool err_handling_test = send_stop || recv_stop;
         unsigned num_iters     = err_handling_test ? 1 : m_num_iters;
 
+        // do small send-recv to check reachability of peers
+        send_recv(sender(), receiver(), SEND_RECV_TAG, false, cb_type());
```

Comment:
doing so beats the purpose of the test - which is to queue operations before the connections is established:
```
        /* send multiple messages to test the protocol both before and after
         * connection establishment */
```

Reply:
yes, I didn't notice it